src/routes/auth.route.ts

```diff
@@ -18,24 +18,29 @@ const authRouter = express.Router();
 // );
 
 authRouter.route("/register").post(async (req: Request, res: Response) => {
-  // try {
-  // process input data
-  // call repository method
-  const data = req.body;
+  try {
+    // process input data
+    // call repository method
+    const data = req.body;
+    const result = await register(data);
+    res.send(result);
+  } catch (error: unknown) {
+    console.error(error);
+    return res.status(400).send(error.toString());
+  }
```

Comment:
```suggestion
  try {
    // process input data
    // call repository method
    const data = req.body;
    const result = await register(data);
    res.send(result);
  } catch (error: unknown) {
    console.error(error);
    return res.status(400).send(error.toString());
  }
```
return, please, block from develop. Exception catches in middlewares